source/user-manual/wazuh-dashboard/management.rst

```diff
@@ -0,0 +1,105 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: The Wazuh dashboard gives you a quick view of your agents, alerts, and cluster. Learn how to configure its features in this section. 
+  
+.. _dashboard_management:
+
+Dashboard management
+========================
+
+The **Dashboard management** section allows you to configure and customize your Wazuh dashboard experience, as well as seeing the generated reports and the application logs.
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Dashboards Management
+---------------------
+
+Index patterns
+^^^^^^^^^^^^^^
+In this section, you can see and configure the Index patterns, and creating new ones.
+
+.. thumbnail:: ../../images/kibana-app/features/settings/index-patterns.png
+  :align: center
+  :width: 100%
+
+Saved objects
+^^^^^^^^^^^^^
+You can see the saved objects of the application here. You will find the index patterns, the application settings and the custom visualizations and dashboards.
+
+.. thumbnail:: ../../images/kibana-app/features/settings/saved-objects.png
+  :align: center
+  :width: 100%
+
+Advanced settings
+^^^^^^^^^^^^^^^^^
+In this section you can configure advanced settings of the Wazuh Dashboard. Here, you can set the appearance to dark mode. 
+
+.. thumbnail:: ../../images/kibana-app/features/settings/advanced-settings.png
+  :align: center
+  :width: 100%
+
+.. thumbnail:: ../../images/kibana-app/features/settings/dark-mode.png
+  :align: center
+  :width: 100%
+
+
+Reporting
+---------
+Here you can see the reports generated in the different modules, with the ``Generate report`` button.
+
+.. thumbnail:: ../../images/kibana-app/features/settings/reporting.png
+  :align: center
+  :width: 100%
+
+Server APIs
+-----------
+
+In this section, you can list all your inserted API credentials. The star icon indicates the currently used API to show information on the app. Each entry has multiple available actions to manage it. Keep in mind that a working API is needed to add or edit an entry. Check your API connection status before adding them to the app.
```

Comment:
This needs a rephrasing. What's "inserted API credentials"? What's "currently used API to show information on the app"? What is a "working API"? Are these entries API connections?

Reply:
Solved in https://github.com/wazuh/wazuh-documentation/pull/7280/commits/055ead35c636ee3ddd651d00f4bddafa07cd4446